pkg/cmd/release-payload-controller/payload_created_controller.go

```diff
@@ -133,9 +150,57 @@ func (c *PayloadCreatedController) sync(ctx context.Context, key string) error {
 		return err
 	}
 
+	failedCondition := conditions.FindCondition(originalReleasePayload.Status.Conditions, v1alpha1.ConditionPayloadFailed)
+	createdCondition := computeReleasePayloadCreatedCondition(originalReleasePayload, failedCondition)
+
 	releasePayload := originalReleasePayload.DeepCopy()
+	conditions.SetCondition(&releasePayload.Status.Conditions, createdCondition)
+	releasepayloadhelpers.CanonicalizeReleasePayloadStatus(releasePayload)
+
+	if reflect.DeepEqual(originalReleasePayload, releasePayload) {
+		return nil
+	}
 
 	klog.V(4).Infof("Syncing Payload Created for ReleasePayload: %s/%s", releasePayload.Namespace, releasePayload.Name)
+	_, err = c.releasePayloadClient.ReleasePayloads(releasePayload.Namespace).UpdateStatus(ctx, releasePayload, metav1.UpdateOptions{})
+	if errors.IsNotFound(err) {
+		return nil
+	}
+	if err != nil {
+		return err
+	}
 
 	return nil
 }
+
+func computeReleasePayloadCreatedCondition(payload *v1alpha1.ReleasePayload, failedCondition *metav1.Condition) metav1.Condition {
+	createdCondition := metav1.Condition{
+		Type:               v1alpha1.ConditionPayloadCreated,
+		Status:             metav1.ConditionUnknown,
+		Reason:             ReleasePayloadCreatedReason,
+		LastTransitionTime: metav1.Now(),
+	}
+
+	if failedCondition != nil && failedCondition.Status == metav1.ConditionTrue {
+		createdCondition.Status = metav1.ConditionFalse
+		createdCondition.Reason = failedCondition.Reason
+		createdCondition.Message = failedCondition.Message
+		return createdCondition
+	}
+
+	// Check that Release Creation job completed successfully...
+	releasePayloadCreateResult := payload.Status.ReleaseCreationJobResult
```

Comment:
there's only ever one of these?  If there's only one, you can directly write the condition.  We used two stages for the prowjobs because there is more than one prowjob we have to inspect.